rally/README.md

```diff
@@ -0,0 +1,68 @@
+# Rally
+
+A Flutter sample app based on the Material study Rally (a hypothetical, personal finance app). It
+showcases custom tabs, custom painted widgets, and custom animations.
+
+For info on the Rally Material Study, see: https://material.io/design/material-studies/rally.html
+
+**This sample is not currently in a finished state. We're in the process
+of building it out.**
+
+## Goals
+
+* Show how to customize a tab bar.
+* Show how to create reusable custom widgets with composition and custom painting.
+* Show how to create an app with tabs and child navigation screens.
+
+## Widgets / APIs
```

Comment:
This is not a section we normally include with our samples. The "important bits" section you've got below is doing the work of explaining in broad terms what people should expect and focus on, so you don't need this list.

Reply:
Done